addons/account/models/partner.py

```diff
@@ -527,6 +532,24 @@ def _compute_duplicated_bank_account_partners_count(self):
         for partner in self:
             partner.duplicated_bank_account_partners_count = len(partner._get_duplicated_bank_accounts())
 
+    @api.depends('credit_limit')
+    @api.depends_context('company')
+    def _compute_credit_limit_compute(self):
+        for record in self:
+            record.credit_limit_compute = record.credit_limit or self.env.company.account_default_credit_limit
```

Comment:
I remember you said that using `0` would be a possibility to mean that a partner hasn't any limit.
If it is in the spec, this won't work

Reply:
Indeed, that is (among other things) what I was referring to when I told you I still had a few things to work on :wink: 
Alongside the `company_dependent` thing that is still not working properly (thus set to `False` at the moment).